From f9b27dee4dc01dafec111957a1ad411f20bcb644 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Fri, 25 Aug 2017 03:17:19 +0900
Subject: [PATCH] Deleted version limit for dosocs2.

Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index b35863f..9be6cd7 100644
--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,6 @@ tests_require=[
 
 setup(
     name='dosocs2',
-    version=_dosocs2_version,
     description='SPDX 2.0 document creation and storage',
     long_description='',
     url='https://github.com/DoSOCSv2/DoSOCSv2',
-- 
2.7.4

