From d282ba074625922d12615af676ac1f0e922db88f Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Wed, 15 Feb 2017 23:23:53 +0900
Subject: [PATCH] setup.py: delete the depends install

Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>
---
 setup.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/setup.py b/setup.py
index 527b161..b35863f 100644
--- a/setup.py
+++ b/setup.py
@@ -3,16 +3,9 @@ from setuptools import setup
 _dosocs2_version = '0.16.1'
 
 install_requires=[
-    'jinja2',
-    'python-magic',
-    'docopt',
-    'SQLAlchemy',
-    'psycopg2'
     ]
 
 tests_require=[
-    'pytest',
-    'mock'
     ]
 
 setup(
-- 
1.8.4.2

